src/components/urls/helpers.tsx

```diff
@@ -60,3 +60,8 @@ export const Copy = ({ text, message, children }: CopyProps) => <BlackLink
     showInfoMessage(message)
   }}
 >{children}</BlackLink>
+
+export const fullPath = (relative: string) => { 
+  const pathname = relative.startsWith('/') ? relative : '/' + relative
```

Comment:
This func should not add `/` in two cases:

1. if base URL already ends with `/`
2. if relative URL already starts with `/`